### PR TITLE
Add last come first served alg

### PR DESCRIPTION
**Pull request by zach401**
Added two simple sorting methods, last_come_first_served and largest_remaining_processing_time.

### CODE REVIEW

File: acnportal/algorithms/sorted_algorithms.py [hunk 1]
```diff
@@ -183,6 +183,17 @@ def first_come_first_served(evs, iface):
     return sorted(evs, key=lambda x: x.arrival)
 
 
+def last_come_first_served(evs, iface):
```
Comment (sunash) [inline acnportal/algorithms/sorted_algorithms.py]: We require a specific call signature for the `sort_fn`'s of (List[EV], Interface) -> List[EV], even when the interface isn't required. Do we want to mention this call signature in the docs?

File: acnportal/algorithms/sorted_algorithms.py [hunk 2]
```diff
@@ -183,6 +183,17 @@ def first_come_first_served(evs, iface):
...
+    """ Sort EVs by arrival time in reverse order.
+    Args:
+       evs (List[EV]): List of EVs to be sorted.
+       iface (Interface): Interface object.
+    Returns:
+       List[EV]: List of EVs sorted by arrival time.
```
Comment (sunash) [inline acnportal/algorithms/sorted_algorithms.py]: (In reverse order)

File: acnportal/algorithms/sorted_algorithms.py [hunk 3]
```diff
@@ -226,4 +237,29 @@ def laxity(ev):
...
 
+def largest_remaining_processing_time(evs, iface):
+    """ Sort EVs by  largest remaining processing time.
+
+    Args:
+        evs (List[EV]): List of EVs to be sorted.
+        iface (Interface): Interface object.
+
+    Returns:
+        List[EV]: List of EVs sorted by departure time.
```
Comment (sunash) [inline acnportal/algorithms/sorted_algorithms.py]: "...by largest remaining processing time."

File: acnportal/algorithms/sorted_algorithms.py [hunk 4]
```diff
@@ -226,4 +237,29 @@ def laxity(ev):
...
+    """
+
+    def remaining_processing_time(ev):
+        """ Calculate laxity of the EV.
```
Comment (sunash) [inline acnportal/algorithms/sorted_algorithms.py]: laxity -> remaining processing time

File: acnportal/algorithms/sorted_algorithms.py [hunk 5]
```diff
@@ -226,4 +237,29 @@ def laxity(ev):
...
+
+        Args:
+            ev (EV): An EV object.
+
+        Returns:
+            float: The laxity of the EV.
```
Comment (sunash) [inline acnportal/algorithms/sorted_algorithms.py]: same as above

File: acnportal/algorithms/sorted_algorithms.py (hunk 1 above)
Comment (zach401) [reply acnportal/algorithms/sorted_algorithms.py]: I added a note in each function where iface is not used.

**Comment by zach401** [comment]
Pushed local version that address most of the commenting concerns.

**Comment by sunash** [comment]
LGTM

**Pull request merged**